src/owner.rs

```diff
@@ -186,9 +186,9 @@ impl Team {
         // FIXME: we just set per_page=100 and don't bother chasing pagination
```

Comment:
So it turns out that the only thing using this function is on line 138 in `pub fn create()`, and that's within this file itself. While you're modifying this function could you remove the `pub` modifier?